src/app/zap-templates/templates/chip/python-chip-ChipCluster.zapt

```diff
@@ -0,0 +1,62 @@
+'''
+{{> header}}
+'''
+
+import ctypes
+from .ChipStack import *
+from .ChipExceptions import *
+
+__all__ = ["ChipCluster"]
```

Comment:
Having a class called ChipCluster that handle only the server side is misleading. Either the name should change to represent the fact that we're only handling server Cluster and Server Cluster commands or we should had the support for the client command and client clusters. Or at least a TODO with an Issue number saying that the client side will be supported in a near future.